lib/stsci/tools/bitmask.py

```diff
@@ -6,8 +6,8 @@
 import numpy as np
 from astropy.utils import deprecated
 
-__version__ = '1.1.1'
-__vdate__ = '30-January-2018'
+__version__ = '1.2.0'
+__vdate__ = '10-August-2021'
 __author__ = 'Mihai Cara'
 
 __all__ = ['interpret_bit_flags', 'bitfield_to_boolean_mask', 'is_bit_flag']
```

Comment:
You probably need to clear the list in `__all__` to fix RTD warnng:

```
WARNING: missing attribute mentioned in :members: or __all__: module stsci.tools.bitmask, attribute is_bit_flag
```

Reply:
Ah, I missed `__all__`

Reply:
Thanks!